samples/matter/light_switch/README.rst

```diff
@@ -155,65 +314,150 @@ The ``build_nrf52840dk_nrf52840`` parameter specifies the output directory for t
       File not found: ./ncs/nrf/samples/matter/light_switch/configuration/nrf52840dk_nrf52840/prj_shell.conf
 
 Testing
-=======
+*******
+
+After building the sample and programming it to your development kit, complete the steps in the following sections.
+
+.. _matter_light_switch_sample_prepare_for_testing:
+
+Prepare for testing
+===================
+
+After building this and the :ref:`Matter Light Bulb <matter_light_bulb_sample>` samples, and programming them to the development kits, complete the following steps:
+
+.. note::
+   In both examples (light switch and light bulb), a Bluetooth LE discriminator is set with the same value by default (in hex 0xF00 in decimal 3840).
+   That means before commissioning, only one uncommissioned device can be powered up; otherwise, chip-tool can commission a random device.
+   If a device becomes commissioned, you can power up next device and perform the subsequent commissioning.
+   To avoid this unclear situation you can set up your unique discriminator in *src/chip_project_config.h* file by changing `CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR` value.
```

Comment:
```suggestion
   To avoid this unclear situation, you can set up your unique discriminator in :file:`src/chip_project_config.h` file by changing :kconfig:option:`CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR` value.
```